services/blockchain-indexer/shared/database/schema/accounts.js

```diff
@@ -17,21 +17,9 @@ module.exports = {
 	primaryKey: 'address',
 	schema: {
 		address: { type: 'string' },
-		publicKey: { type: 'string' },
-		isGenesisAccount: { type: 'boolean', null: false, defaultValue: false },
-		isDelegate: { type: 'boolean', null: false, defaultValue: false },
-		balance: { type: 'bigInteger' },
-		username: { type: 'string' },
-		rewards: { type: 'bigInteger', defaultValue: 0 },
-		producedBlocks: { type: 'integer', defaultValue: 0 },
-		totalVotesReceived: { type: 'string' },
-	},
-	indexes: {
-		address: { type: 'key' },
-		publicKey: { type: 'key' },
-		isDelegate: { type: 'key' },
-		balance: { type: 'range' },
-		username: { type: 'key' },
+		isGenerator: { type: 'boolean', null: false, defaultValue: false },
```

Comment:
Please rename `isGenerator` to `isValidator` everywhere.
```suggestion
		isValidator: { type: 'boolean', null: false, defaultValue: false },
```